package.yaml

```diff
@@ -62,3 +62,18 @@ tests:
       - tasty-golden
       - wai-saml2
       - xml-conduit
+  validation:
+    main: validation.hs
+    source-dirs: tests
+    ghc-options: -Wall -Wcompat
+    dependencies:
+      - base
+      - bytestring
+      - filepath
+      - pretty-show
+      - tasty
+      - tasty-expected-failure
+      - tasty-hunit
+      - transformers
+      - wai-saml2
+      - xml-conduit
```

Comment:
I am curious about your decision to have this in an entirely separate test suite. What are your thoughts on having one test suite which combines all the tests with `tasty`?

Reply:
That's a good question, I now think having an unified executable is better so I'll combine these tomorrow.